governance/src/components/Unstaking/UnstakingModalPreview.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useMemo } from "react";
+import styled from "styled-components";
+
+import {
+  BaseModalContentColumn,
+  SecondaryText,
+  Subtitle,
+  Title,
+} from "shared/lib/designSystem";
+import colors from "shared/lib/designSystem/colors";
+import { UnstakeIcon } from "shared/lib/assets/icons/icons";
+import moment from "moment";
+import { useRBNTokenAccount } from "shared/lib/hooks/useRBNTokenSubgraph";
+import useTextAnimation from "shared/lib/hooks/useTextAnimation";
+import { formatBigNumber } from "shared/lib/utils/math";
+import { ActionButton } from "shared/lib/components/Common/buttons";
+
+const LogoContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 64px;
+  height: 64px;
+  background: ${colors.red}1F;
+  border-radius: 100px;
+`;
+
+interface UnstakingModalPreviewProps {
+  onUnstake: () => void;
+}
+
+const UnstakingModalPreview: React.FC<UnstakingModalPreviewProps> = ({
+  onUnstake,
+}) => {
+  const { data: rbnTokenAccount, loading } = useRBNTokenAccount();
+  const loadingText = useTextAnimation(loading);
+
+  const canUnstake = useMemo(() => {
+    if (loading || !rbnTokenAccount || !rbnTokenAccount.lockEndTimestamp) {
+      return false;
+    }
+
+    return moment().isSameOrAfter(
+      moment.unix(rbnTokenAccount.lockEndTimestamp)
+    );
+  }, [loading, rbnTokenAccount]);
+
+  const lockupAmountDisplay = useMemo(() => {
+    if (loading) {
+      return loadingText;
+    }
+
+    return `${
+      rbnTokenAccount?.lockedBalance
+        ? formatBigNumber(rbnTokenAccount?.lockedBalance)
```

Comment:
Suggestion for typing: `rbnTokenAccount && rbnTokenAccount.lockedBalance ? formatBigNumber(rbnTokenAccount.lockedBalance)`